SecurityService.OpenIdConnect.IntegrationTests/Common/DockerHelper.cs

```diff
@@ -5,38 +5,158 @@
+            }
+
+            // Now build and return the container                
+            IContainerService builtContainer = securityServiceContainer.Build().Start().WaitForPort($"{port}/tcp", 30000);
+            Thread.Sleep(20000); // This hack is in till health checks implemented :|
+
+            AddEntryToHostsFile("127.0.0.1", containerName);
+
+            logger.LogInformation("Security Service Container Started");
+
+            return builtContainer;
+        }
+
+        private static IContainerService SetupSecurityServiceTestUIContainer(String containerName, 
+                                                                String securityServiceContainerName, 
+                                                                Int32 securityServiceContainerPort,
+                                                                INetworkService networkService,
+                                                                (String clientId, String clientSecret) clientDetails)
+        {
+            // Management API Container
+            IContainerService securityServiceTestUIContainer = new Builder().UseContainer().WithName(containerName)
+                                                                            .WithEnvironment($"Authority=http://{securityServiceContainerName}:{securityServiceContainerPort}",
+                                                                                             $"ClientId={clientDetails.clientId}",
+                                                                                             $"ClientSecret={clientDetails.clientSecret}")//,
+                                                                                             //$"MetadataAddress=http://{securityServiceContainerName}:{securityServiceContainerPort}/.well-known/openid-configuration")
+                                                                            .UseImage("securityservicetestwebclient").ExposePort(5004)
+                                                                            .UseNetwork(new List<INetworkService>
+                                                                                        {
+                                                                                            networkService
+                                                                                        }.ToArray())
+                                                                            .Build().Start().WaitForPort("5004/tcp", 30000);
+
+            return securityServiceTestUIContainer;
+
+            //Console.Out.WriteLine("Started Security Service");
+        }
+
+        private readonly NlogLogger Logger;
+        protected List<INetworkService> TestNetworks;
+        protected List<IContainerService> Containers;
         public Guid TestId;
-        private void SetupTestNetwork()
+        public String SecurityServiceContainerName;
```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/stuart_ferguson1/SecurityService/pullRequest?prid=6144581)